modules/apps/sharing/sharing-test/src/testIntegration/java/com/liferay/sharing/service/test/SharingEntryLocalServiceTest.java

```diff
@@ -562,6 +562,7 @@ public void testDeleteGroupSharingEntries() throws Exception {
 	public void testDeleteGroupSharingEntriesDoesNotDeleteOtherGroupSharingEntries()
 		throws Exception {
 
+		Group group1 = _group;
```

Comment:
Hi @brianchandotcom,

In this case I prefer using `group1` and `group2` instead of `_group` and `group` to be able to differentiate between `group1SharingEntries` and `group1SharingEntries` in the code below. In the rest of the cases I've applied your pattern.

Thanks!